src/js/hca/components/HealthCareApp.jsx

```diff
@@ -1,3 +1,5 @@
+/* global __API_URL__ */
```

Comment:
Ooh..nifty. I didn't know about that.

I'm going to migrate that back into the .eslintrc though so globals are in one spot. Will do in follow-up PR.